pkg/tfvars/azure/azure.go

```diff
@@ -84,7 +82,6 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 		Environment:                     environment,
 		ARMEndpoint:                     sources.ARMEndpoint,
 		Region:                          region,
-		BootstrapInstanceType:           defaults.BootstrapInstanceType(sources.CloudName, region),
```

Comment:
Can the `BootstrapInstanceType` function be removed?

Reply:
It's not being used anywhere now so removing it :+1: